docs/source/conf.py

```diff
@@ -75,7 +75,7 @@
 exclude_patterns = ['_build']
 
 # The reST default role (used for this markup: `text`) to use for all documents.
-# default_role = None
+default_role = 'any'
```

Comment:
What does this do? Just curious.

Reply:
It defines the default link directive as `any`, so every time you do something like
```
`text`
```
In anything (docstrings, sphinx docs, etc.) it tries to find a valid link target somewhere in your documentation, whether it be a page, a class, a label, etc.

E.g. you don't have to put :directives: in your docstrings to link to functions

Reply:
😮